helix-core/src/test/java/org/apache/helix/manager/zk/TestZkBaseDataAccessor.java

```diff
@@ -34,11 +37,24 @@
 import org.apache.helix.manager.zk.ZkBaseDataAccessor.RetCode;
 import org.apache.zookeeper.data.Stat;
 import org.testng.Assert;
-import org.testng.annotations.AfterClass;
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.Test;
 
 public class TestZkBaseDataAccessor extends ZkUnitTestBase {
+  private static final ZkSerializer CUSTOM_ZK_SERIALIZER = new ZkSerializer() {
```

Comment:
Add a quick comment saying what this custom serializer does?

"A serializer that takes a String and converts it to a byte array."